srcpkgs/lsd/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'lsd'
+pkgname=lsd
+version=0.16.0
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config"
+#makedepends="rust cargo"
+depends="awesome-terminal-fonts"
+short_desc="The next gen ls command"
+maintainer="Lugubris <lugubris@disroot.org>"
+license="Apache-2.0"
+homepage="https://github.com/Peltoche/${pkgname}"
```

Comment:
please don't use variables in the homepage field. This makes it harder to just copy the URL